app/src/main/java/org/blitzortung/android/data/MainDataHandler.kt

```diff
@@ -356,7 +356,8 @@ class MainDataHandler @Inject constructor(
 
             Mode.ANIMATION -> {
                 parameters = parameters.animationStep(history)
-                val delay = if (parameters.isRealtime()) 3000L else 150L
+                val delay = 150L //if (parameters.isRealtime()) 3000L else 150L
+                // There is no need pause because it is possible to study the situation when the animation is turned off.
```

Comment:
But it would be great to have the current situation frozen for a short moment. This could be configurable via preferences, so that everybody can freely decicde.

Reply:
The decision is yours, as the author of the application. I would like to see the animation without delay, and in the settings I would choose this option. This is just my suggestion, which may be wrong. :)

Reply:
Will make that configurable so that everyone can choose.